rcl_yaml_param_parser/src/parser.c

```diff
@@ -936,7 +936,8 @@ static rcutils_ret_t parse_value(
 
   if (val_size > MAX_STRING_SIZE) {
     RCUTILS_SET_ERROR_MSG_WITH_FORMAT_STRING(
-      "Scalar value at line %d is bigger than %d bytes", line_num, MAX_STRING_SIZE);
+      "Scalar value at line %d has %zu bytes which is bigger than the compile "
```

Comment:
Should the `%d` for `line_num` be `%u`, since it is unsigned (uint32_t)?

Reply:
Since I didn't change that part of the message I didn't consider touching it. I am also not aware of any warnings on our CI builds for it.

Reply:
Fair enough.

Reply:
CI seems to be happy with both. Since 716d50baed1c1036473beda62f65acb47f357bb5 makes a bit more sense with %u I added it anyway.